src/HatchIntakeSubsystem.cpp

```diff
@@ -12,7 +12,7 @@ HatchIntakeSubsystem::HatchIntakeSubsystem() : m_hatchIntake(HATCH_INTAKE_WHEEL_
 {
 }
 
-void HatchIntakeSubsystem::robotInit(){
+void HatchIntakeSubsystem::robotInit() {
     operatorJoystick->RegisterButton(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON);
     operatorJoystick->RegisterButton(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER);
     
```

Comment:
I don't understand why you are trying to set the neutral mode of each of these talons, and why you set the modes to two different modes.